app/views/pages/_modal-art.html.erb

```diff
@@ -0,0 +1,72 @@
+ <div class="container justify-content-center my-4">
+
+    <div class="container">
+
+        <div class="container art-piece-grid">
+          <% @user.art_pieces.each do |art_piece| %>
+        <div class="card-art-piece">
+
+          <% if art_piece.photos.attached? %>
+            <%= cl_image_tag art_piece.photos[0].key, crop: :fill, class:"img-index" %>
+          <% else %>
+            <img class="img-index" src="https://images.unsplash.com/photo-1581783342308-f792dbdd27c5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1834&q=80"/>
+          <% end %>
+
+          <div class="card-art-piece-infos">
+          
+            <div>
+
+              <h2><%= art_piece.title %></h2>
+              <p><%= art_piece.artist %>.</p>
+
+                <div class="functional-buttons">
+                  <h2><%= link_to 'Delete', art_piece, method: :delete, data: { confirm: 'Are you sure?' } %></h2>
+                  <h2><%= link_to 'Edit', art_piece, {:class => 'tooltips', 'data-toggle' => 'modal', 'data-target' => '#exampleModal'+art_piece.id.to_s}%></h2>
+                </div>
+
+            </div>
+
+          <h2 class="card-art-piece-pricing"><%= "#{art_piece.price} € | mth" %></h2>
+
+          </div>
+
+          <%= link_to "", art_piece_path(art_piece), class: "card-link" %>
+          </div>
+
+          <div class="modal fade" id="exampleModal<%=art_piece.id%>" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h5 class="modal-title" id="exampleModalLabel">Editing <b><%= art_piece.title %></b></h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                  </button>
+                </div>
+                <div class="modal-body">
+                  <%= simple_form_for(art_piece) do |f| %>
+                    <%= f.error_notification %>
+                    <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
+                    <div class="form-inputs">
+                      <%= f.input :title %>
+                      <%= f.input :artist %>
+                      <%= f.input :price %>
+                      <%= f.input :description %>
```

Comment:
If the user would like to edit his/ her art-piece, it would be nice if he/ she can also upload another picture. What do you think? 
<%= f.input :photos, as: :file, input_html: { multiple: true }, hint: 'please upload high-resolution pictures' %>